Code/GraphMol/SmilesParse/CXSmilesOps.cpp

```diff
@@ -467,15 +478,7 @@ bool parse_coordinate_bonds(Iterator &first, Iterator last, RDKit::RWMol &mol,
     if (read_int_pair(first, last, aidx, bidx)) {
       if (VALID_ATIDX(aidx) && bidx >= startBondIdx &&
           bidx < startBondIdx + mol.getNumBonds()) {
```

Comment:
I know this isn't part of your PR, but I think this could be:
```
      if (VALID_ATIDX(aidx) && VALID_BNDIDX(bondIdx)) {
```

Reply:
Ok, I'll update it.